src/test/java/redis/clients/jedis/tests/JedisClusterTest.java

```diff
@@ -606,4 +629,22 @@ private boolean isAnyNodeHandshaking(Jedis node) {
     }
     return false;
   }
+
+  @Test
+  public void testBitfield() {
```

Comment:
@smadasu you don't need to create another request, remove the unnecessary tests, push the stuff to your branch and we can squash the commits later.